packages/terra-date-input/src/DateInput.jsx

```diff
@@ -102,6 +102,10 @@ const propTypes = {
    * Custom input attributes to apply to the year input
    */
   yearAttributes: PropTypes.object,
+  /**
+   * Custom props to identify the error message node
+   */
+  customProps: PropTypes.Object,
```

Comment:
we don't need declaration for customProps

Reply:
Removed.